Packages/StatusKit/Sources/StatusKit/Row/Subviews/StatusRowMediaPreviewView.swift

```diff
@@ -245,6 +246,29 @@ struct BlurOverLay: View {
     default: false
     }
   }
+  
+  // return either labelColor or primaryBackgroundColor, whichever contrasts better against
+  // the specified background color
+  private func contrastingColor(against backgroundColor: Color) -> Color {
```

Comment:
This should probably live in Theme and be cached, this is a bit performance consuming to have this in the view